### ISSUE TITLE
CircleCI Integration tests job failing

### ISSUE THREAD

**Issue by jrbenny35** [mozilla/addons-server#9570]
Currently the integration tests are failing on Circleci. The job looks like it is passing as it is returning a Code 2.

```
Running setup.py install for pygit2 ... - \ error
    Complete output from command /code/.tox/ui-tests/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-Ll6Cfj/pygit2/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-p8kOiu/install-record.txt --single-version-externally-managed --compile --install-headers /code/.tox/ui-tests/include/site/python2.7/pygit2:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-2.7
    creating build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/py3.py -> build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/submodule.py -> build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/_build.py -> build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/__init__.py -> build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/config.py -> build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/blame.py -> build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/utils.py -> build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/settings.py -> build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/index.py -> build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/_run.py -> build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/repository.py -> build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/errors.py -> build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/remote.py -> build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/ffi.py -> build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/py2.py -> build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/credentials.py -> build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/refspec.py -> build/lib.linux-x86_64-2.7/pygit2
    copying pygit2/decl.h -> build/lib.linux-x86_64-2.7/pygit2
    warning: build_py: byte-compiling is disabled, skipping.
    
    running build_ext
    generating cffi module 'build/temp.linux-x86_64-2.7/pygit2._libgit2.c'
    creating build/temp.linux-x86_64-2.7
    building '_pygit2' extension
    creating build/temp.linux-x86_64-2.7/src
    x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fno-strict-aliasing -Wdate-time -D_FORTIFY_SOURCE=2 -g -fstack-protector-strong -Wformat -Werror=format-security -fPIC -I/usr/local/include -I/usr/include/python2.7 -c src/blob.c -o build/temp.linux-x86_64-2.7/src/blob.o
    In file included from src/blob.c:30:0:
    src/blob.h:33:18: fatal error: git2.h: No such file or directory
    compilation terminated.
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
    
    ----------------------------------------
Command "/code/.tox/ui-tests/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-Ll6Cfj/pygit2/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-p8kOiu/install-record.txt --single-version-externally-managed --compile --install-headers /code/.tox/ui-tests/include/site/python2.7/pygit2" failed with error code 1 in /tmp/pip-install-Ll6Cfj/pygit2/
Makefile-docker:110: recipe for target 'install_python_test_dependencies' failed
make: *** [install_python_test_dependencies] Error 1
WARNING: command failed but result from testenv is ignored
cmd: InvocationError for command '/usr/bin/make -f Makefile-docker update_deps' (exited with code 2)
ui-tests runtests: commands[1] | make -f Makefile-docker run-ui-tests
rm -rf ./user-media/* ./tmp/*
# Reset the database and fake database migrations
python manage.py reset_db --noinput
/code/settings.py:140: UserWarning: Could not import local_settings module. Traceback (most recent call last):
  File "/code/settings.py", line 134, in <module>
    from local_settings import *  # noqa
ImportError: No module named local_settings

  traceback.format_exc()))
Traceback (most recent call last):
  File "manage.py", line 11, in <module>
    execute_from_command_line(sys.argv)
  File "/code/.tox/ui-tests/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/code/.tox/ui-tests/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 338, in execute
    django.setup()
  File "/code/.tox/ui-tests/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/code/.tox/ui-tests/local/lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models()
  File "/code/.tox/ui-tests/local/lib/python2.7/site-packages/django/apps/config.py", line 202, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/code/src/olympia/abuse/models.py", line 6, in <module>
    from olympia.addons.models import Addon
  File "/code/src/olympia/addons/models.py", line 46, in <module>
    from olympia.files.models import File
  File "/code/src/olympia/files/models.py", line 38, in <module>
    from olympia.files.utils import SafeZip, write_crx_as_xpi
  File "/code/src/olympia/files/utils.py", line 30, in <module>
    import flufl.lock
ImportError: No module named flufl.lock
Makefile-docker:151: recipe for target 'setup-ui-tests' failed
make: *** [setup-ui-tests] Error 1
WARNING: command failed but result from testenv is ignored
cmd: InvocationError for command '/usr/bin/make -f Makefile-docker run-ui-tests' (exited with code 2)
```

**Comment by jrbenny35** [comment]
Updated this with the actual error. It seems pygit2 isn't installing so the other dependencies aren't either.

**Comment by jrbenny35** [comment]
Fixed in #9624

**Comment by AlexandraMoga** [comment]
Please add a qa:not needed label to this issue.
Thanks
